content/news/2025-04-04-fosdem-recordings.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Recordings for Railways and Open Transport Devroom at FOSDEM available
+date: 2025-04-04
+image:
+  src: images/news/FOSDEM_logo.svg.png
```

Comment:
In the last years we've added a picture of the packed devroom. I feel it would be a bit more friendly instead of using a logo that at least half of our readership doesn't know.

Reply:
Agree to the first (everything else would be disappointing), and let's try the second ;)